ush/python/pygw/src/pygw/fsutils.py

```diff
@@ -33,7 +33,7 @@ def chdir(path):
         os.chdir(path)
         yield
     finally:
-        print(f"WARNING: Unable to chdir({path})")  # TODO: use logging
+        print(f"WARNING: Unable to chdir({path})")
```

Comment:
but you're not using logging here, right? So keep the TODO?

Reply:
right. may be I should keep the `#TODO`.